contact.html

```diff
@@ -1,22 +1,52 @@
 <!DOCTYPE html>
```

Comment:
@zakkweb @cynthia9

I wrote explanations in some of these code lines, to show some CSS I figured out